sdk/core/azure-core/azure/core/tracing/__init__.py

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
-from ._abstract_span import AbstractSpan, SpanKind, HttpSpanMixin
+from azure.core.tracing._abstract_span import AbstractSpan, SpanKind, HttpSpanMixin
```

Comment:
Please import from azure.core.tracing instead (they are all available there, without the need to import a private module)

Reply:
this init is in the azure.core.tracing.

Reply:
Laurent meant not to import from "_abstract_span"

Reply:
yes, but this import is to expose them publicly from the private module.

Reply:
will do that fix in a second PR